docs/flow/modin/core/execution/ray/implementations/pandas_on_ray/index.rst

```diff
@@ -1,8 +1,34 @@
+PandasOnRay Execution
+=====================
+
+Queries that perform data transformation, data ingress or data egress using the `pandas on Ray` execution
+pass through the Modin components detailed below.
+
+To enable the `pandas on Ray` execution, please refer to the usage section in :doc:`pandas on Ray </UsingPandasonRay/index>`.
+
+Data Transformation
+'''''''''''''''''''
+
+.. image:: /img/pandas_on_ray_data_transform.svg
+   :align: center
+
+When a user calls any :py:class:`~modin.pandas.dataframe.DataFrame` API, a query starts forming at the `API` layer
+to be executed at the `Execution` layer. The `API` layer is responsible for processing the query appropriately,
+for example, determining whether the final result should be a ``DataFrame`` or ``Series`` object, and sanitizing the input to the
```

Comment:
Shouldn't there be links here to `DataFrame` and `Series`?

Reply:
That doesn't make a lot of sense. We refer to DataFrame and Series as general objects here.